climpred/bootstrap.py

```diff
@@ -332,6 +335,8 @@ def bootstrap_compute(
                         ensemble. Choose from:
                         [:py:func:`bootstrap_uninitialized_ensemble`,
                          :py:func:`bootstrap_uninit_pm_ensemble_from_control`].
+        baseline_compute (func): function to compute a baseline forecast skill with.
```

Comment:
(1) We should standardize the language to "reference" instead of "baseline." I changed "ref"/"reference" in our code base to "verif" since reference is reserved in the community for a "reference forecast". I.e., persistence, uninitialized (forced), climatology, random mechanism.

(2) This should probably just accept a string from a specific list of strings. Right now our only reference option is persistence.

So maybe make `baseline_compute` instead `reference` and default to `'persistence'`. You could have a constant called `REFERENCE_FORECASTS` that contains `['persistence']` and do a check that the argument is in that list. This will give us the infrastructure for adding future references.

Reply:
(1) I use baseline because in the docs we also call it baseline. then lets change the docs page also.

(2) now with callables/funcs it accepts a function that a user can easily prototype without change the source code